setup.sh

```diff
@@ -40,6 +42,10 @@ if [[ $LIBTPU_GCS_PATH == NONE ]]; then
   unset LIBTPU_GCS_PATH
 fi
 
+if [[ $SKIP_REQUIREMENTS == NONE ]]; then
+    SKIP_REQUIREMENTS="false"
```

Comment:
This code pattern doesn't look correct to me.  Because it leaves a state where SKIP_REQUIREMENTS can either be false or unset which seems undesirable? (The code does seem OK but it weird we expose so many options.)

Reply:
But I like the feature a lot.

Reply:
done, following the same pattern as other args!